text/processing.py

```diff
@@ -282,7 +282,7 @@ class WriteToTextFileJob(Job):
 
     def __init__(self, content, out_name: str = "file.txt"):
         """
-        :param list|dict|str content: input which will be written into a text file
+        :param list|dict|str|DelayedBase content: input which will be written into a text file
```

Comment:
Since you are touching the code, could you move the typing to the header (like for `out_name`)?